src/Application/ConfigLookup.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseFacetedSearch\Application;
```

Comment:
The interfaces go within Application/, their persistence-mechanism-specific implementations go into Persistence/. 

(It is possible to have interfaces end up in Persistence/ if they are used by a class in there, in which case they won't be used in Application/)

Reply:
The interfaces form the architectural boundary

Reply:
Blame WikibaseExport developers!